docs/general/community-standards.md

```diff
@@ -5,48 +5,53 @@ title: Community Standards
 
-- If the issue cannot be resolved between the contributors, any complaints, instances of explicit rulebreaking, or unresolvable disagreements with other community members may be directed towards the Administrative team, or the Project Leaders directly. You may do so through email (`team [at] jellyfin.org`) or on Matrix via a direct message. Please include details and context as appropriate.
+- If the issue cannot be resolved between the contributors, any complaints, instances of explicit rulebreaking, or unresolvable disagreements with other community members may be directed towards the Core Team or the Project Leader directly. You may do so through email (`team [at] jellyfin.org`) or on Matrix via a direct message. Please include details and context as appropriate. See the [about page](xref:about) for the list of Core Team members and ways to contact them.
 
 - The team will review the complaint and decide on an action, including but not limited to: informal private guidance, informal private warnings, formal public warnings, temporary ban(s) from the various platforms, or permanent ban(s).
 
 - Formal warnings will be made under a "second-chance-only" policy. Once warned, if one repeats the same behaviour, the response will escalate as appropriate to the infraction.
 
-- Moderation tasks in some locations are delegated to other team members. This dispute resolution procedure applies anywhere under the Jellyfin umbrella. The Administrative team and Project Leaders retain final say in any dispute resolutions.
+- Moderation tasks in some locations are delegated to other, non-Core team members. This dispute resolution procedure applies anywhere under the Jellyfin umbrella. The Core team and Project Leader retain final say in any dispute resolutions.
+
+## Questions or Comments
+
+Questions or comments regarding these standards should be forwarded to the [Project Leader](https://matrix.to/#/@joshuaboniface:bonifacelabs.ca) or [Core Team](xref:about).
 
 ## Changelog
 
 This document represents official Jellyfin project policy. Any changes to this document require a changelog entry here and approval by a Project Leader.
 
-2020-09-14, Joshua Boniface: Initial version of the community standards document. Based _very_ loosely on several CoCs including the Contributor Covenant, and various Forum rules I've read and written over the years.
+2020-09-14, Joshua Boniface: Initial version of the community standards document. Based *very* loosely on several CoCs including the Contributor Covenant, and various Forum rules I've read and written over the years.
+2022-09-03, Joshua Boniface: Update header and footer sections; reduce redundant wording; make rules clearer.
```

Comment:
Did you check if this shows correctly in the site? In the GitHub markdown preview it doesn't have a line break.

Reply:
I just checked and both lines are combined as a single paragraph.

Reply:
Interesting when I did my local docfx build it looked right. I'll add a `<br/>`.

Reply:
Actually, used bullet points instead.

Reply:
Probably a difference in how docfx and docusaurus handle markdown.